test/ssl-tests/15-certstatus.cnf

```diff
@@ -1,9 +1,11 @@
 # Generated with generate_ssl_tests.pl
 
-num_tests = 2
```

Comment:
No, this file must be kept.

Reply:
Interesting - looks like there is general room for improvement regarding these files
since, usually, auto-generated files should not be put in a repo but re-produced when needed.